server/data/data-cassandra/src/test/java/org/apache/james/sieve/cassandra/CassandraSieveDAOTest.java

```diff
@@ -105,10 +113,11 @@ void insertScriptShouldUpdateActivate() {
     void deleteScriptInCassandraShouldWork() {
         sieveDAO.insertScript(USERNAME, SCRIPT).block();
 
-        sieveDAO.deleteScriptInCassandra(USERNAME, SCRIPT_NAME).block();
-
-        Optional<Script> actual = sieveDAO.getScript(USERNAME, SCRIPT_NAME).blockOptional();
-        assertThat(actual).isEmpty();
+        CALMLY_AWAIT.atMost(Durations.TEN_SECONDS)
+            .untilAsserted(() -> {
+                sieveDAO.deleteScriptInCassandra(USERNAME, SCRIPT_NAME).block();
+                assertThat(sieveDAO.getScript(USERNAME, SCRIPT_NAME).blockOptional()).isEmpty();
+            });
```

Comment:
Could we know why in the first place the deletion is not instantaneous?

Reply:
as my comment above

> It looks like from a lightweight transaction at deleteScriptStatement when contain .ifExists()

Reply:
I tried to remove `.ifExists`, and this error never come

Reply:
Actually thinking more about this, LWTs are used instead of indempotence, it would thus make sense to drop this IMO.

(And disable non-idenpotent tests)